submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,237 @@
+const url = 'https://randomuser.me/api/?results=12';
+const form = document.forms;
+const content = document.querySelector('.articleBlock');
+
+let friendsCopy = [];
+let friends = [];
+let isSearch = false;
+
+async function getResponse(url) {
+    let response;
+    try {
+        response = await fetch(url);
+        if (!response.ok) {
+            throw new Error(response.statusText);
+        } 
+    } catch (err) {
+        console.log(err);
+        resetPage();
+    }
+    let jsonData = await response.json();
+
+    friendsCopy = jsonData.results;
+    friends = jsonData.results;
+
+    renderAllItemsToPage(friends);
+}
+
+function creatingDivItem(item) {
+    return `
+        <div class='content__item'>
+            <img class='content__item-img' src='${item.picture.medium}'>
+            <div class='content__item-name'>${item.name.title} ${item.name.first} ${item.name.last}</div>
+            <p class='content__item-email'>
+                <a href="mailto:  ${item.email}"   target="_blank"> ${item.email}</a>
+            </p>
+            <p>Age: ${item.dob.age}</p>
+            <a href='${item.phone}'>${item.phone}</a>
+            <p class='content__item-country'>${item.location.country}, ${item.location.city}</p>
+            <div class='content__item-gender'>${item.gender}</div>
+        </div>
+    `;
+}
+
+form.last.addEventListener('click', function(e) {
+    if (e.target && e.target.classList.contains('last')) {
+        checkLastSort(e);
+    }
+});
+
+function checkLastSort(event) {
+    form.age.reset();
+    switch(event.target.value) {
+        case "lastAToZ":
+            sortedAtoZLast();
+            break;
+        case "lastZToA":
+            sortedZtoALast();
+            break;
+    }
+}
+
+form.reset.reset.addEventListener('click', resetPage);
```

Comment:
Consider resetting the form state and rerendering users instead of reloading the whole page. You can also use [this](https://developer.mozilla.org/ru/docs/Web/API/HTMLFormElement/reset).